Twig/Extension/DeprecatedTextExtension.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Twig\Extension;
+
+/**
+ * NEXT_MAJOR : remove this class and the twig/extensions dependency.
```

Comment:
Just started a search through the code ;) Haven't found any usage for `NEXT_MAJOR` and `@deprecated`.

Don't have the ticket where we discussed this.